solutions/scripts/lib-franklin.js

```diff
@@ -124,7 +124,7 @@ export async function loadCSS(href) {
  * @param {Object} attrs additional optional attributes
  */
 
-export async function loadScript(src, attrs) {
+export async function loadScript(src, attrs = null) {
```

Comment:
I would avoid changing core functions of `lib-franklin` because there is a risk that it will be overwritten, once we update `lib-franklin` to a new version.